docs/features.md

```diff
@@ -125,6 +125,8 @@ As you can see, the edge between `develop` and `call-ws` changed to grey &mdash;
 You can easily rename a branch directly from Git Machete.
 Open a context menu for a branch and choose `Rename...` action.
 
+For the tracked branches, it is also possible to rename the remote branch.
```

Comment:
@PawelLipski , can you pls piggyback a fix here? To reflect [this](https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1497#discussion_r1127556124) better (currently it is very inaccurate). sorry I have not fixed it in the first place

Reply:
#1611